utilities/src/main/java/io/syndesis/qe/resource/impl/SFTP.java

```diff
@@ -0,0 +1,126 @@
+        OpenShiftUtils.getInstance().serviceAccounts()
+            .createNew()
+            .withNewMetadata()
+            .withName(SERVICE_ACCOUNT_NAME)
+            .endMetadata()
+            .done();
+        OpenShiftUtils.getInstance()
+            .addRoleToUser("anyuid", "system:serviceaccount:" + TestConfiguration.openShiftNamespace() + ":" + SERVICE_ACCOUNT_NAME);
+
+        if (!TestUtils.isDcDeployed(APP_NAME)) {
+            List<ContainerPort> ports = new LinkedList<>();
+            ports.add(new ContainerPortBuilder()
+                .withName("sftp-cmd")
+                .withContainerPort(SFTP_PORT)
+                .withProtocol("TCP").build());
+
+            List<EnvVar> templateParams = new ArrayList<>();
+            templateParams.add(new EnvVar("SFTP_USERS", USER_AND_PASSWORD, null));
+            OpenShiftUtils.getInstance().deploymentConfigs().createOrReplaceWithNew()
+                .editOrNewMetadata()
+                .withName(APP_NAME)
+                .addToLabels(LABEL_NAME, APP_NAME)
+                .endMetadata()
+
+                .editOrNewSpec()
+                .addToSelector(LABEL_NAME, APP_NAME)
+                .withReplicas(1)
+                .editOrNewTemplate()
+                .editOrNewMetadata()
+                .addToLabels(LABEL_NAME, APP_NAME)
+                .endMetadata()
+                .editOrNewSpec()
+                .addNewContainer().withName(APP_NAME).withImage("syndesisqe/sftpd:latest")
+                .addAllToPorts(ports)
+                .addAllToEnv(templateParams)
+
+                .withNewSecurityContext()
+                .endSecurityContext()
+                .endContainer()
+                .endSpec()
+                .endTemplate()
+                .addNewTrigger()
+                .withType("ConfigChange")
+                .endTrigger()
+                .endSpec()
+                .done();
+
+            ServiceSpecBuilder serviceSpecBuilder = new ServiceSpecBuilder().addToSelector(LABEL_NAME, APP_NAME);
+
+            serviceSpecBuilder.addToPorts(new ServicePortBuilder()
+                .withName("sftp-cmd")
+                .withPort(SFTP_PORT)
+                .withTargetPort(new IntOrString(SFTP_PORT))
+                .build());
+
+            OpenShiftUtils.getInstance().services().createOrReplaceWithNew()
+                .editOrNewMetadata()
+                .withName(APP_NAME)
+                .addToLabels(LABEL_NAME, APP_NAME)
+                .endMetadata()
+                .editOrNewSpecLike(serviceSpecBuilder.build())
+                .endSpec()
+                .done();
+
+            try {
+                OpenShiftWaitUtils.waitFor(OpenShiftWaitUtils.areExactlyNPodsReady(LABEL_NAME, APP_NAME, 1));
+                Thread.sleep(20 * 1000);
+            } catch (InterruptedException | TimeoutException e) {
+                log.error("Wait for {} deployment failed ", APP_NAME, e);
+            }
+        }
```

Comment:
this should not be necessary as it is handled by https://github.com/syndesisio/syndesis-qe/blob/master/utilities/src/main/java/io/syndesis/qe/resource/ResourceFactory.java#L26